### ISSUE TITLE
Provide a way to configure secrets to download private Docker images

### ISSUE THREAD

**Issue by drobiazko** [openintegrationhub/openintegrationhub#23]
This is not critical yet as we can work with open Docker images in the beginning but this becomes critical for private components

**Comment by anton-kotenko** [comment]
Options:
1. make end user to create docker secret. Add it into flow definition. Bad: in increases user-to-platform interface
2. add credentials into flow definition crd. Create docker secret internally by flow-operator

Personally i prefer 2 option. 

In both cases it's required to find way to protect secret from usage from other flows. (service account per flow + k8s RBAC?)

**Comment by philecs** [comment]
@anton-kotenko  any updates here?

**Comment by anton-kotenko** [comment]
Hi @philecs 
There are no updates, because this task is blocked by #32 and partially #28 And now i'm working on those blocker tasks.

**Comment by drobiazko** [comment]
During implementation we realized that we might need to revise our decision on using k8s CRDs for flows. It's probably better to postpone the implementation of this issue until we have a final decision, probably in Sprint 2.

**Comment by philecs** [comment]
@anton-kotenko and @drobiazko do we have some updates here?

**Comment by anton-kotenko** [comment]
@philecs 
No updated, at least from me.

**Comment by RobinBrinkmann** [comment]
Backlog clean-up. Closing of outdated issues.
